app/controllers/api/base_controller/renderer.rb

```diff
@@ -569,6 +569,18 @@ def render_create_resource_options(ems_id)
         render_options(type, :form_schema => klass.params_for_create(ems))
       end
 
+      def render_attach_resource_options(id)
+        type = @req.collection.to_sym
+        resource = resource_search(id, type)
+        raise BadRequestError, resource.unsupported_reason(:attach_volume) unless resource.supports?(:attach_volume)
+
+        if resource.try(:params_for_attach) ## TODO can we make this one line?
+          render_options(type, :form_schema => resource.params_for_attach)  
+        else
+          render_options(type, :form_schema => {}) ## {:fields => []}
+        end
+      end
+
```

Comment:
Hey @MelsHyrule I thought for this simplified version we were going to drop the OPTIONS / params_for_attach and take care of that in a follow-up

Reply:
This function slipped through the cracks when i was simplifying, it's not being used or called so ill remove it now